x-pack/plugins/alerting/server/rules_client_factory.ts

```diff
@@ -47,8 +45,6 @@ export interface RulesClientFactoryOpts {
   minimumScheduleInterval: AlertingRulesConfig['minimumScheduleInterval'];
   maxScheduledPerMinute: AlertingRulesConfig['maxScheduledPerMinute'];
   connectorAdapterRegistry: ConnectorAdapterRegistry;
-  getAlertIndicesAlias: GetAlertIndicesAlias;
-  alertsService: AlertsService | null;
```

Comment:
Lost why did we delete these 2 methods from here?

Reply:
Again a mistake when fixing conflicts 🙂. Thanks! Fixed in [`71898c9` (#167884)](https://github.com/elastic/kibana/pull/167884/commits/71898c9a25f6a521b111437ba978876d4aa9379f)